cmd/minishift/cmd/start.go

```diff
@@ -98,6 +98,13 @@ var (
 		Usage:     "Registry mirrors to pass to the Docker daemon.",
 		Value:     cmdUtil.NewStringSliceValue([]string{}, &[]string{}),
 	}
+
+	nameServersFlag = &flag.Flag{
+		Name:      configCmd.NameServers.Name,
+		Shorthand: "",
+		Usage:     "Specify nameserver to use for the instance.",
+		Value:     cmdUtil.NewStringSliceValue([]string{}, &[]string{}),
+	}
```

Comment:
It seems this flag is exposed for all base operating systems. But other flags are only exposed for Windows. Any specific reason?
```

	if runtime.GOOS == "windows" {
		startFlagSet.String(configCmd.NetworkDevice.Name, "eth0", "Specify the network device to use for the IP address. Ignored if no IP address specified (Hyper-V only)")
		startFlagSet.String(configCmd.IPAddress.Name, "", "Specify IP address to assign to the instance (Hyper-V only)")
		startFlagSet.String(configCmd.Netmask.Name, "24", "Specify netmask to use for the IP address. Ignored if no IP address specified (Hyper-V only)")
		startFlagSet.String(configCmd.Gateway.Name, "", "Specify gateway to use for the instance. Ignored if no IP address specified (Hyper-V only)")
	}
```

Reply:
Setting the IP address only works on Windows... and this will be until we modify drivers. Since we will have most issues with the KVM driver, we would rather see the move to the new organisation happen.